### PR TITLE
Create environment.yml

### LINKED ISSUE #22: ruyaml conda dependency
I have worked out an `environment.yaml` for the conda recipe version. There is an issue with the `ruyaml` dependency as a result of the fork from `ruamel.yaml`. 

The conda version is available from the latter but not from the former (yet). 

I am not sure if we can revert to `ruamel.yaml` or using the fork is important but we can elevate the issue upstream if needed.

### PR DESCRIPTION
**Pull request by brey**
Add conda env
Move to ruamel.yaml for conda compatibility
Pin llvmlite for OSX compatibility
Avoid python 3.10 for llvmlite conf
Pin black

resolves #22

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -45,14 +45,15 @@ bokeh = "^2.2"
...
 datashader = "^0.13"
-ruyaml = "^0.20.0"
+"ruamel.yaml" = "^0.17.0"
 geoviews = "^1.9.2"
 Cartopy = "<0.20"  # due to https://github.com/holoviz/geoviews/issues/529
 Shapely = "<1.8"
 
 [tool.poetry.dev-dependencies]
-black = "^21.11b1"
+black = "^21.11b0"
```
Comment (pmav99) [inline pyproject.toml:56]: What is the problem with 22.11b1? Or is it not available in conda-forge?

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+dependencies:
+  - python=3.9
+  - proj<8
+  - datashader
+  - panel
+  - holoviews
+  - geoviews
+  - shapely=1.7.1
+  - ruamel.yaml
+  - black=21.11b0
```
Comment (pmav99) [inline environment.yml:18]: Cartopy > 0.20 causes bad performance. I guess that conda pulls cartopy 0.19 due to `proj<8` but I would rather have it as a primary dependency, too. We do `import cartopy` after all.

**Review by pmav99 — CHANGES_REQUESTED** [review]
Thanks @brey, just a couple of nitpicks

File: pyproject.toml (hunk 1 above)
Comment (brey) [reply pyproject.toml:56]: Indeed this is latest on conda-forge at least for OSX

File: environment.yml (hunk 2 above)
Comment (brey) [reply environment.yml:18]: Done

**Review by pmav99 — APPROVED** [review]

**Pull request merged**